enterprise/frontend/src/embedding-sdk/lib/theme/default-component-theme.ts

```diff
@@ -138,5 +138,62 @@ export function getEmbeddingComponentOverrides(
         ...(theme?.popover?.zIndex && { zIndex: theme.popover.zIndex }),
       },
     },
+    // TODO: fix those, they're not all correct
+    ModalRoot: {
+      defaultProps: {
+        withinPortal: true,
+        // portalProps: { target: `#${EMBEDDING_SDK_ROOT_ELEMENT_ID}` },
+        target: `#${EMBEDDING_SDK_ROOT_ELEMENT_ID}`,
+      }, // satisfies Partial<ModalRootProps>,
```

Comment:
This is "fun", mantine doesn't typecheck default props for performance reason and suggests to use `sastifies xxx` if we want to typecheck default props.
Our version of prettier is old enough to fail if we use `satisfies`, i'll try and open a pr to update it